tools/pipelines/Variables.yml

```diff
@@ -0,0 +1,13 @@
+variables:
+  - name: FORCE_COLOR
+    value: 1
+  - template: /tools/pipelines/templates/include-vars.yml@self
+    parameters:
+      publishOverride: '${{ parameters.publishOverride }}'
+      releaseBuildOverride: '${{ parameters.releaseBuildOverride }}'
+      buildNumberInPatch: '${{ parameters.buildNumberInPatch }}'
+      nonScopedPackages: '${{ parameters.nonScopedPackages }}'
+  - name: absolutePathToTelemetryGenerator
+    value: $(Build.SourcesDirectory)/tools/telemetry-generator
+    readonly: true
+  - group: ado-feeds
```

Comment:
Why is this pulled out into a separate file?

Reply:
It's an automated tool that did the conversion, probably just it trying to be smart.

Reply:
From what I can tell the 1es stuff is built on top of ADO's base capabilities, so I think the existing way we include the vars file should work:

```yaml
variables:
  - name: FORCE_COLOR
    value: 1
  - template: include-vars.yml
    parameters:
      publishOverride: ${{ parameters.publishOverride }}
      releaseBuildOverride: ${{ parameters.releaseBuildOverride }}
      buildNumberInPatch: ${{ parameters.buildNumberInPatch }}
      nonScopedPackages: ${{ parameters.nonScopedPackages }}
  - name: absolutePathToTelemetryGenerator
    value:  $(Build.SourcesDirectory)/tools/telemetry-generator
    readonly: true
  - group: ado-feeds
```